components/automate-ui/src/app/pages/project/list/project-list.component.spec.ts

```diff
@@ -559,43 +559,118 @@ describe('ProjectListComponent', () => {
   });
 
   describe('getButtonText', () => {
```

Comment:
Re-organized this test section (and added a couple new tests) to more closely parallel the new section below it.